packages/databricks-vscode/package.json

```diff
@@ -394,47 +394,47 @@
                 },
                 {
                     "command": "databricks.connection.bundle.selectTarget",
-                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.target.*$/",
+                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.target.*$/ && databricks.context.bundle.deploymentState == idle",
                     "group": "inline@2"
                 },
                 {
                     "command": "databricks.connection.configureLogin",
-                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.authType.*$/",
+                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.authType.*$/ && databricks.context.bundle.deploymentState == idle",
                     "group": "inline@2"
                 },
                 {
                     "command": "databricks.connection.attachClusterQuickPick",
-                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.cluster.*$/",
+                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.cluster.*$/ && databricks.context.bundle.deploymentState == idle",
                     "group": "inline@2"
                 },
                 {
                     "command": "databricks.cluster.stop",
-                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.cluster.(running|pending)$/",
+                    "when": "view == configurationView && viewItem =~ /^databricks.configuration.cluster.(running|pending)$/ && databricks.context.bundle.deploymentState == idle",
                     "group": "inline@0"
                 },
                 {
                     "command": "databricks.cluster.start",
-                    "when": "view == configurationView && viewItem == databricks.configuration.cluster.terminated",
+                    "when": "view == configurationView && viewItem == databricks.configuration.cluster.terminated && databricks.context.bundle.deploymentState == idle",
                     "group": "inline@0"
                 },
                 {
                     "command": "databricks.bundle.deployAndRun",
-                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.runnable.*$/",
+                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.runnable.*$/ && databricks.context.bundle.deploymentState == idle",
                     "group": "inline@0"
                 },
                 {
                     "command": "databricks.bundle.cancelRun",
-                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.cancellable.*$/",
+                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.cancellable.*$/ && databricks.context.bundle.deploymentState == running",
```

Comment:
`deploying`

 Maybe we can just have `bundle.deploying` flag? Or do we expect to have more than two deployment states?

Reply:
Possibly error for enabling force?

Reply:
True. Also, the first part of my comment was about incorrect `running` state, is should be `deploying`, right?

Reply:
Aah. Copilot 😅

Reply:
https://github.com/databricks/databricks-vscode/pull/1064